update-instructions.md

```diff
@@ -30,6 +30,10 @@ If you added or removed some mods:
 8. Modpack will now update.
 9. Run it and check the version difference in the bottom right corner.
 
+### Modrinth App
+
+TODO
```

Comment:
someone can continue this

Reply:
So only this is missing right?

Reply:
For now, yeah
Not sure what else might change before the release